core/scripts/local/getMedianHistoricalPrice.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * @notice This is an example script demonstrating how to get a historical median price at a specific timestamp.
+ * It could be used for example to query a price before committing a vote for a DVM price request.
+ * We provide default configurations for querying median prices on specific markets across some exchanges and markets.
+ * This script should serve as a template for constructing other historical median queries.
+ *  *
+ * @dev How to run: $(npm bin)/truffle exec ./scripts/local/GetMedianHistoricalPrice.js --network mainnet --identifier <PRICE-FEED IDENTIFIER> --time <TIMESTAMP IN SECONDS>
+ */
+const { fromWei } = web3.utils;
+const { createPriceFeed } = require("../../../financial-templates-lib/price-feed/CreatePriceFeed");
+const { Networker } = require("../../../financial-templates-lib/price-feed/Networker");
+const winston = require("winston");
+const argv = require("minimist")(process.argv.slice(), { string: ["identifier", "time"] });
+
+// Pricefeed default configurations to pass into Medianizer price feed. Medianizer returns the median price across specified
+// `medianizedFeeds`.
+const defaultConfigs = {
+  "ETH/BTC": {
+    type: "medianizer",
+    pair: "ethbtc",
+    lookback: 604800,
+    minTimeBetweenUpdates: 60,
+    medianizedFeeds: [
+      { type: "cryptowatch", exchange: "coinbase-pro" },
+      { type: "cryptowatch", exchange: "binance" },
+      { type: "cryptowatch", exchange: "bitstamp" }
+    ]
+  },
+  COMPUSD: {
+    type: "medianizer",
+    lookback: 604800,
+    minTimeBetweenUpdates: 60,
+    medianizedFeeds: [
+      { type: "cryptowatch", exchange: "coinbase-pro", pair: "compusd" },
+      { type: "cryptowatch", exchange: "poloniex", pair: "compusdt" },
+      { type: "cryptowatch", exchange: "ftx", pair: "compusd" }
+    ]
+  }
+};
```

Comment:
Why not use https://github.com/UMAprotocol/protocol/blob/061ccbceeb75d03d641a03780885f04ba24b0489/financial-templates-lib/price-feed/CreatePriceFeed.js#L188 ? If you want to override the lookback, you can do so by passing in `{ lookback: 604800}` as the last parameter. 

We should've probably called that parameter `configOverride` in hindsight.

Reply:
I didn't want to have to pass in the EMP address. I wanted this script to be EMP independent, but maybe we shouldn't do that.

Reply:
Oh I see. That's a good point. Can we refactor that library to have 2 functions then, one that takes the EMP address and one that takes an identifier? They can be really simple wrappers around a function that holds the default configs. The only reason I suggest that is the default configs and the behavior you want is really similar to that code -- so some very minor tweaks could make it work for this use case.

Reply:
Done. `createReferencePriceFeedForEmp` now takes in an optional `identifier` param which must be specified if `empAddress` is missing.